src/Widgets/NavigationWidget/SubComponents/Brand.tsx

```diff
@@ -15,7 +17,7 @@ export const Brand = connect(function Brand({
           content={root}
           attribute="siteLogoDark"
           className="navbar-brand-logo"
-          alt="Logo"
+          alt={alt}
```

Comment:
* [ ] Why don't we use the `siteLogoDark`'s alternative text? This way, editors only have to set the alt text once.
* [ ] FYI checking this I realized we have an unused `siteLogoLight` setting.

Reply:
siteLogoLight will be addressed in a different PR.